src/raft/raw_node.rs

```diff
@@ -357,3 +357,35 @@ impl<T: Storage> RawNode<T> {
         self.raft.mut_store()
     }
 }
+
+#[cfg(test)]
+mod test {
+    use kvproto::eraftpb::MessageType;
+    use super::is_local_msg;
+
+    #[test]
+    fn test_is_local_msg() {
+        let tests = vec![
+        (MessageType::MsgHup, true),
```

Comment:
It's odd that rustfmt would not format this as expected. The rustfmt version is `0.6.0 ()`.